.circleci/config.yml

```diff
@@ -1,33 +1,59 @@
 version: 2.1
 
 executors:
-  node:
+  node-10:
     docker:
       - image: circleci/node:10
+  node-12:
+    docker:
+      - image: circleci/node:12
 workflows:
   version: 2
   build-and-test:
     jobs:
-      - build
+      - build:
+          name: build12
+          e: node-12
+      - build:
+          name: build10
+          e: node-10
+      - test-dev-tools:
+          e: node-12
+          requires:
+            - build12
       - test-dev-tools:
+          e: node-10
           requires:
-            - build
+            - build10
       - test-json-file:
+          e: node-12
+          requires:
+            - build12
+      - test-json-file:
+          e: node-10
+          requires:
+            - build10
+      - test-expo-codemod:
+          e: node-12
           requires:
-            - build
+            - build12
```

Comment:
TIL CircleCI doesn't support build matrixes, bummer. 😕

Anyway, it's important that we do this, even if it introduces some accidental complexity to the CI configuration 👍